lib/internal/process/modules.js

```diff
@@ -19,7 +23,20 @@ function normalizeReferrerURL(referrer) {
 }
 
 function initializeImportMetaObject(wrap, meta) {
-  meta.url = wrap.url;
+  const vmModule = wrapToModuleMap.get(wrap);
```

Comment:
could just simplify this to `const initalizeImportFunction = wrapToFnMap.get(wrap)`?

Reply:
How does const initializeImportMeta = wrapToFnMap.get(wrap) serve as a simplification?